Backend/Microservices/AuthService/src/Auth.Application/DI/Seed.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Auth.BLL.DI
+{
+    public static class Seed
+    {
+        public static async Task InitializeRoles(IServiceProvider serviceProvider)
+        {
+            using var scope = serviceProvider.CreateScope();
+            var roleManager = scope.ServiceProvider.GetRequiredService<RoleManager<IdentityRole<long>>>();
+            var roles = new[] { "Admin", "User", "Moder" };
```

Comment:
Use the constants from RoleConsts instead of hardcoding role names ("Admin", "User", "Moder") in InitializeRoles